### PR TITLE
Static setup for pages

### LINKED ISSUE #16: EDIT: Set up static pages (OLD: Begin work on landing and variable page)
Implement code box, subtitle and robots, no functionality however (so if a user clicks a robot nothing needs to happen). The "python.tsx" file may give you an idea of how to approach this especially with using "CodeFormat.tsx". My thought here was making separate tsx files for the page and one scss for the both to share however if you have a better idea go for it!

### PR DESCRIPTION
**Pull request by archishadatta**
All the pages formatted except experiment page.
CSS fixed for sidebar.
Closes #16

### CODE REVIEW

File: src/components/pages/advanced.tsx
Comment (nendow02) [inline src/components/pages/advanced.tsx]: It might be better to make these buttons since users are going to have to select them. This goes for all pages except "output".

File: src/components/pages/variable.tsx
Comment (nendow02) [inline src/components/pages/variable.tsx]: Just noticed that string values are in white, would blue make more sense since number and bool values are blue and variable names are white, wondering what your thoughts are.

File: src/components/pages/advanced.tsx
Comment (nendow02) [inline src/components/pages/advanced.tsx]: Can we take out` id="landing"`? This goes for all the other pages that have it too.

File: src/components/pages/variable.tsx
Comment (nendow02) [inline src/components/pages/variable.tsx:1]: Can we delete `variable.scss` file since it’s not used anymore.

**Review by nendow02 — CHANGES_REQUESTED** [review]
Great job! I just have a few suggestions

File: src/components/pages/variable.tsx
Comment (archishadatta) [reply src/components/pages/variable.tsx]: I'm going off the protocol on [https://www.onlinegdb.com/](https://www.onlinegdb.com/). I'm assuming it's because strings are not primitives while number/bool values are. But if you think it might confuse the kids, I can change it your call.

**Comment by archishadatta** [comment]
@nendow02 Fixed the issues. Not sure if @reginawang99 needs to review as well before merging.

**Review by nendow02 — APPROVED** [review]

**Comment by nendow02** [comment]
Approved! I also took care of merge conflicts, you may want to look over it though just to make sure none of the CSS was messed up. Since next button has disable functionality you can change pages by editing the URL.

**Comment by nendow02** [comment]
@reginawang99 a review would be great, but I know you have a lot of pull requests to go over so if you can’t make it to this one it’s not the end of the world, the primary changes are css.

**Comment by reginawang3495** [comment]
@ldalton02 wanna take a look? (i can take a very cursory glance after)

File: src/styles/levelSelect.scss
Comment (ldalton02) [inline src/styles/levelSelect.scss]: duplicate align-self

**Review by ldalton02 — APPROVED** [review]
Meant to approve! Looks good to me, the changes to level-select and sidebar display might cause a conflict with changes I made but I'll deal with them before I push again

File: src/styles/sidebar.scss
Comment (reginawang3495) [inline src/styles/sidebar.scss]:
```suggestion
```

File: src/styles/python.scss
Comment (reginawang3495) [inline src/styles/python.scss]:
```suggestion
```

File: src/components/shared/sidebar.tsx
Comment (reginawang3495) [inline src/components/shared/sidebar.tsx]: not in the scope of this pr, but jsut for cleanup sake, tbh i had to dig a little before i realized `desc` referred to `description`--can we use the word `description` instead?

File: src/components/pages/variable.tsx
Comment (reginawang3495) [inline src/components/pages/variable.tsx]:
```suggestion
```

File: src/components/pages/python.tsx
Comment (reginawang3495) [inline src/components/pages/python.tsx]:
```suggestion
```

File: src/components/pages/advanced.tsx
Comment (reginawang3495) [inline src/components/pages/advanced.tsx:48]: `<div id='code'>` probs belongs in `CodeFormat`

File: src/components/pages/advanced.tsx
Comment (reginawang3495) [inline src/components/pages/advanced.tsx:63]: id recommend having a `Robot` component if you have time to separate it out! This robot component will deal with all the importing of images, and could be passed in something like
```
interface RobotProps {
   color: Color;
   headShape: Shape;
   hasMouth: boolean;
   numAntennas: number;
   numArms: number;
   numButtons: number;
}
```

This might take a bit of time since you would need to separate the robots by its attributes. I'd recommend having a `robots` folder where robots are named something like `robot-RCM121` and having a `README.md` that briefly mentions 
``` 
file names are based on the robot attibutes
robot-CS[M]###
C refers to robot color
S refers to robot shape
M refers to whether there is a mouth. no M means no mouth
#1 is # of antennas
#2 is # of arms
#3 is # of buttons
```

Comment (reginawang3495) [reply src/components/pages/advanced.tsx:63]: The reason why this is better is because it makes it easier to change the exact details of the project in addition to making this more reusable in the future. In addition, it's more obviously visible what's going on in the code <which is very important since code is read much more than it is written>

Hard coding things works, but since we are doing react and not html/css it's good to leverage the react benefits!! I know you already did it this way, so I don't mind keeping it this way since it's not a small effort to change to this. Totes your choice

File: src/components/pages/output.tsx
Comment (reginawang3495) [inline src/components/pages/output.tsx]: can we make a separate `CheckedDropdowns` component which deals with `onCorrect` and is passed in the expected correct values in the end
```suggestion
            <div id='dropdowns'>
              {Object.keys(dropValues).map((dropdown) => (
                <BooleanDropdown key={dropdown} onChange={(value) => selected(value,dropdown)} position={dropdown}/>
              ))}
            </div>

          </div>
```

**Review by reginawang3495 — CHANGES_REQUESTED** [review]
looks just like the fimga 🎈  waow its coming together hehe

In general, when doing React <and most CS minus HTML/CSS lol>, if you find yourself copy and pasting a lot<and you care about code cleanliness>, it's good to see if there are ways to reduce duplication. Duplication causes code bloat and makes things harder to maintain. It's just like how Figma itself has components so that you can modify one central area.

also, would you mind removing some duplicated lines? its really nbd, just a nit :)

File: src/components/pages/output.tsx
Comment (archishadatta) [reply src/components/pages/output.tsx]: @nendow02 Think this is your stuff

File: src/components/pages/advanced.tsx
Comment (archishadatta) [reply src/components/pages/advanced.tsx:48]: hey @reginawang99 this would break the css because id="code" refers to the entire CodeFormat component

File: src/components/pages/advanced.tsx
Comment (archishadatta) [reply src/components/pages/advanced.tsx:63]: @nendow02 What do you think would you like me to do this? It would take me a while probably, I'm not really familiar with interfaces and stuff in React. But it would be more reusable.

Comment (nendow02) [reply src/components/pages/advanced.tsx:63]: I think we should save this for later. The other method would be more ideal but seeing that the way we have it now is perfectly functional I think its fine to leave it this way for now.

File: src/components/pages/output.tsx
Comment (nendow02) [reply src/components/pages/output.tsx]: Just pushed a change for this!

**Review by reginawang3495 — APPROVED** [review]
good enough for me :)

**Pull request merged**
